### ISSUE TITLE
[wavereform] Tests fail when using latest 2016 GCD file (Trac #1896)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1888]
FAIL: testChi (__main__.SanityCheck)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "wavereform/resources/test/wavereform_basics.py", line 150, in testChi
    self.assert_(all(x < 1e4 for x in chi[om]), "Chi for good pulses is below threshold")
AssertionError: Chi for good pulses is below threshold

----------------------------------------------------------------------

Migrated from https://code.icecube.wisc.edu/ticket/1896
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:24", 
    "description": "FAIL: testChi (__main__.SanityCheck)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"wavereform/resources/test/wavereform_basics.py\", line 150, in testChi\n    self.assert_(all(x < 1e4 for x in chi[om]), \"Chi for good pulses is below threshold\")\nAssertionError: Chi for good pulses is below threshold\n\n----------------------------------------------------------------------\n", 
    "reporter": "olivas", 
    "cc": "jbraun", 
    "resolution": "fixed", 
    "_ts": "1550067204154158", 
    "component": "cmake", 
    "summary": "[wavereform] Tests fail when using latest 2016 GCD file", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-10-13T21:25:49", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161103T19:37:59: 
* jbraun commented:

> In changeset 151313:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="151313"
> Multiple issues:  Add filters so we can run wavereform over the entire detector, using the 2016 GCD file; ensure all DOMs get at least one launch: borked DOMs with no launch have OK chi2; reduce the time shift so that borked FADC SLC pulses are still in the 3-bin window.  Closes #1888.
> ```


* jbraun changed resolution from "" to "fixed"
* jbraun changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:24: 
* olivas changed milestone from "October 2016" to ""
* olivas commented:

> Milestone October 2016 deleted
